src/main/java/address/preferences/PrefsManager.java

```diff
@@ -0,0 +1,91 @@
+package address.preferences;
```

Comment:
If you name the class `PrefManager`, name the package `pref`, to be consistent with other packages.